README.md

```diff
@@ -5,7 +5,7 @@
 
 A collection of lints to catch common mistakes and improve your [Rust](https://github.com/rust-lang/rust) code.
 
-[There are over 700 lints included in this crate!](https://rust-lang.github.io/rust-clippy/master/index.html)
+[There are over 750 lints included in this crate!](https://rust-lang.github.io/rust-clippy/master/index.html)
```

Comment:
Should this (and the similar line in `book/src/README.md`) be captured in this patch?

Reply:
Wdym by captured? `cargo dev update_lints` automatically updates this and fails CI if this lint count is not up to date.

Reply:
Ok, I didn't know that the CI would fail. I would consider this a problem though, as several proposed PR contain this mandatory change right now, while others don't and might need it if they are to be merged first.

Wouldn't it be better if this was updated at release time rather than when adding a lint? (and yes, this is unrelated to this particular PR)

Reply:
I don't think it's as much of a problem. Whichever PR gets merged first "wins" and all other PRs can just drop their change during the rebase. New lint PRs getting merged often create merge conflicts on most other lint PRs anyway so you often have to rebase either way.
Old PRs that don't have the lint count updated but need it will still fail when `r+`'d as bors makes sure that CI still passes with the PR integrated into master, and fixing it is very easy with `cargo dev update_lints`.

It also happens rather infrequently (last change on that line was 10 months ago) that I don't know if it's worth complicating this process (this number replacement is [very little code](https://github.com/rust-lang/rust-clippy/blob/9cf416dc6ec9d7ebc8df299ca970e4a92efa2596/clippy_dev/src/update_lints.rs#L50-L58) in `update_lints` which already handles all other kinds of content generation).

But if you want to discuss it more, I'd suggest opening a thread on Zulip